server/routes/juror-management/juror-record/juror-record.controller.js

```diff
@@ -1304,22 +1296,16 @@
 
   function canSummon(req, commonDetails) {
     const jurorStatus = resolveJurorStatus(commonDetails);
-    let canSummon = true;
-
-    if (commonDetails.owner !== '400' && !isCourtUser(req)) {
-      canSummon = false;
-    };
 
     switch (jurorStatus) {
     case 'Undeliverable':
     case 'Responded':
     case 'Completed':
     case 'Disqualified':
```

Comment:
do we need to add excused to the list?

Reply:
That would need confirmation, if we defer someone without entering a response... would then, we never being able to enter a response correct?

Reply:
updated